test/test_jit.py

```diff
@@ -4246,16 +4243,14 @@ def foo(self, input):
         with self.assertRaisesRegex(RuntimeError, 'called recursively involving'):
             M()
 
-    # TODO: Use this when we support passing literals to script fns
-    @unittest.expectedFailure
     def test_script_kwargs_fn_call(self):
         class M(torch.jit.ScriptModule):
             def __init__(self):
                 pass
 
             @torch.jit.script_method
             def call_foo(self, input):
-                return self.foo(input=input, bar=1)
+                return self.foo(input=input, bar=_to_tensor(1))
```

Comment:
Actually I think the idea here was to pass `bar` as an int. This needs a type annotation below.